File: src/main/java/org/oewntk/model/Tracing.java
/*
 * Copyright (c) 2021. Bernard Bou.
 */

package org.oewntk.model;

import java.io.OutputStream;
import java.io.PrintStream;

public class Tracing
{
	public static final PrintStream psInfo = System.out;

	public static final PrintStream psErr = System.err;

	public static final PrintStream psNull = new PrintStream(new OutputStream()
		{
			@Override
			public void write(final int i)
			{
				// do nothing
			}
		});
}

File: src/main/java/org/oewntk/model/Tracing.kt
/*
 * Copyright (c) 2021. Bernard Bou.
 */
package org.oewntk.model

import java.io.OutputStream
import java.io.PrintStream

object Tracing {

    val psInfo: PrintStream = System.out

    val psErr: PrintStream = System.err

    val psNull: PrintStream = PrintStream(object : OutputStream(
    ) {
        override fun write(i: Int) {
            // do nothing
        }
    })
}
